source/portable/80x86/linux_st/gnu/rkht.h

```diff
@@ -46,14 +46,18 @@
 /* -------------------------------- Authors -------------------------------- */
 /*
  *  LeFr  Leandro Francucci  lf@vortexmakes.com
- *  DaBa  Dario Bali�a       dariosb@gmail.com
+ *  DaBa  Dario Baliña       dariosb@gmail.com
+ *  CaMa  Carlos Mancón		 manconci@gmail.com
```

Comment:
You should use spaces instead of tabs

Reply:
My bad. Corrected on https://github.com/vortexmakes/RKH/commit/501d4e6a4647ab0dc8a5c1de563eaaa9a65f307d